FORMS.md

```diff
@@ -19,7 +19,7 @@ These methods use the attributes of a `FormSpec` instance to configure their beh
   Flag signaling whether to strip content in brackets (**and** strip leading and trailing whitespace)
 - `replacements`: `[(' ', '_'), ('̀m', 'm')]`
   List of pairs (`source`, `target`) used to replace occurrences of `source` in formswith `target` (before stripping content in brackets)
-- `first_form_only`: `True`
+- `first_form_only`: `False`
```

Comment:
This attribute of a `FormSpec` controls whether multiple forms will be read from a single Value field. To capture the spelling variants in the CLDF, we need to keep all forms.